v2v-helper/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 	var envPassword = os.Getenv("VCENTER_PASSWORD")
 	var envInsecure = os.Getenv("VCENTER_INSECURE")
 	var sourcevmname = os.Getenv("SOURCE_VM_NAME")
-	var networkname = os.Getenv("NEUTRON_NETWORK_NAME")
+	var networknames = os.Getenv("NEUTRON_NETWORK_NAME")
```

Comment:
I think this should be plural `NEUTRON_NETWORK_NAMES`

Reply:
Fixed